website_blog_more_post_types/README.rst

```diff
@@ -0,0 +1,19 @@
+image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Hi @gfcapalbo 
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst